docs/content/node-operation/node-setup.mdx

```diff
@@ -16,9 +16,12 @@ The hardware your Node will need varies depending on the role your Node will pla
 | **Execution**    | 64 cores  | 800 GB |  9 TB  | n2-highmem-128 |
 | **Verification** |  2 cores  | 16 GB  | 200 GB | n2-highmem-2   |
 | **Access**       |  16 cores | 64 GB  | 900 GB | n2-standard-16 |
+| **Observer**     |  2 cores  | 4 GB   | 300 GB | n2-highmem-16  |
```

Comment:
just double checking: is n2-highmem-16 the right machine for an observer node? Looking at GCP, it seems n2-standard-4 (or similar) is a better fit. Seems strange that Observer has a more powerful machine that Access, right?

Reply:
ah good catch!! i think we overprovisioned our observer node and I blindly copied the instance type.